src/detect-engine-register.h

```diff
@@ -86,6 +83,20 @@ enum DetectKeywordId {
     DETECT_BYTETEST,
     DETECT_BYTEJUMP,
     DETECT_BYTEMATH,
+    DETECT_BYTE_EXTRACT,
+    DETECT_DATASET,
+    DETECT_DATAREP,
+    DETECT_BASE64_DECODE,
+    DETECT_BASE64_DATA,
+    DETECT_BSIZE,
```

Comment:
Should `DETECT_ISDATAAT` be in here as well ?

Reply:
And `DETECT_AL_URILEN` cf https://github.com/OISF/suricata/pull/10334/files#diff-20b12e868dc376bbab11d9bd32507cc49db149d15f3d2274c0697e4700351bd5

Reply:
yes